yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -471,9 +505,6 @@ private[spark] class Client(
       val renewalInterval = getTokenRenewalInterval(stagingDirPath)
       sparkConf.set("spark.yarn.token.renewal.interval", renewalInterval.toString)
     }
-    // Set the environment variables to be passed on to the executors.
-    distCacheMgr.setDistFilesEnv(env)
-    distCacheMgr.setDistArchivesEnv(env)
```

Comment:
I noticed that we order some of these code blocks around in this patch. Is there a particular reason for it? Have you verified that it's safe to do so?